language-server/README.md

```diff
@@ -0,0 +1,11 @@
+Here is pre-built [vscode-css-languageserver](https://github.com/vscode-langservers/vscode-css-languageserver).
+
+
+# Build
+
+Just run `compile-language-server.sh`. The built result will be in `out/`.
```

Comment:
> Can you make it clear that this is not something user needs to do? We should specify that this is something only plugin developers need to do. Maybe we should have # Package Development header?

OK.

> BTW. Is out/ the correct path? I think it's missing language-server/.

I think the path is correct since it's not the top-level README. It is the one in https://github.com/sublimelsp/LSP-css/tree/f6c4705d8b123189675ed1093038e4cae80227ef/vscode-css.